packages/storybook8/stories/Components/SendBox/RichTextSendBox/snippets/RichTextSendBoxAttachmentUploads.snippet.tsx

```diff
@@ -1,6 +1,12 @@
-import { FluentThemeProvider, RichTextSendBox } from '@azure/communication-react';
+import { FluentThemeProvider, RichTextSendBox, DEFAULT_COMPONENT_ICONS } from '@azure/communication-react';
+import { initializeIcons, registerIcons } from '@fluentui/react';
+import { initializeFileTypeIcons } from '@fluentui/react-file-type-icons';
 import React from 'react';
 
+initializeIcons();
+initializeFileTypeIcons();
+registerIcons({ icons: { ...DEFAULT_COMPONENT_ICONS } });
```

Comment:
Do we need same comment here as your other PR?

![image](https://github.com/user-attachments/assets/03a6d0e7-a1a6-4771-a704-009c0cd2cd79)

Reply:
Not in this case. This is just in the concept pages so this code would not be shown to readers